ContractsApi/V1/Boundary/Requests/Validation/CreateContractRequestObjectValidator.cs

```diff
@@ -0,0 +1,17 @@
+using FluentValidation;
+using System;
+
+namespace ContractsApi.V1.Boundary.Requests.Validation
+{
+    public class CreateContractRequestObjectValidator : AbstractValidator<CreateContractRequestObject>
+    {
+        public CreateContractRequestObjectValidator()
```

Comment:
Would be good to implement the rest of the business rules here too. For example `BlockContract` cannot have `ParentContractId`. This ensures data always follows the current rules.